app/src/main/java/protect/card_locker/ImportURIHelper.java

```diff
@@ -38,11 +41,15 @@ public LoyaltyCard parse(Uri uri) throws InvalidObjectException {
             // These values are allowed to be null
             Integer headerColor = null;
             Integer headerTextColor = null;
+            Integer starred;
 
             String store = uri.getQueryParameter(STORE);
             String note = uri.getQueryParameter(NOTE);
             String cardId = uri.getQueryParameter(CARD_ID);
             String barcodeType = uri.getQueryParameter(BARCODE_TYPE);
+            if (uri.getBooleanQueryParameter(STARRED, false) == false) starred = 0;
+                else starred = 1;
```

Comment:
thanks, agreed!
In the end when starred is going to SQLite I need an int since SQLite does not have a separate Boolean storage class. Option could be to only use int instead of boolean?

Reply:
not relevant any longer since starStatus is not imported